nautobot/docs/release-notes/version-1.4.md

```diff
@@ -50,10 +50,14 @@ A new configuration setting, [`STRICT_FILTERING`](../configuration/optional-sett
 
 ### Removed
 
+### Added
+
 ## v1.4.0a2 (2022-MM-DD)
 
 ### Added
 
+- [#1000](https://github.com/nautobot/nautobot/issues/1000) - Object detail views can now have extra UI tabs which are defined by a plugin.
```

Comment:
As a feature, this may be worth documenting in more depth under `## Release Overview` -> `### Added` section as well?